Jenkinsfile

```diff
@@ -16,7 +16,7 @@ node {
     stage('Installing Packages') {
       sh("rm -rf ./venv")
       sh("virtualenv --no-site-packages ./venv")
-      sh("./bin/install-dependencies.sh")
+      sh("./bin/install-dependencies.sh ./venv/bin/pip")
```

Comment:
Is this script only ran by Jenkins or are we running it in our VM too?

Reply:
The `install-dependencies` script is run by Jenkins, on the VM, and when deploying to production.  The `Jenkinsfile` is only run by Jenkins.

Reply:
oh... I see why we may have a few problems with pip in the VM then. I remember using Ansible in the past and the way to install packages is not to run an external script but I reckon we may want to reuse the same script for the VM, for Jenkins etc...